# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../crm114/parallelism.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Makefile.orig	2005-12-02 01:21:01.000000000 -0300
+++ ./Makefile	2005-12-02 01:21:58.000000000 -0300
@@ -310,13 +310,13 @@
 		crm_util_errorhandlers.o \
 		crm_osbf_maintenance.o -lm -ltre -o osbf-util
 
-cssdiff: cssdiff.c crm_util_errorhandlers.o $(HFILES)
+cssdiff: cssdiff.c crm_str_funcs.o crm_util_errorhandlers.o $(HFILES)
 	$(CC)  $(CFLAGS) -D$(SYSTEMTYPE) -DVERSION='"$(VERSION)"' -c cssdiff.c -o cssdiff.o
 	$(CC)  $(CFLAGS) $(LDFLAGS) \
 		crm_util_errorhandlers.o \
 		crm_str_funcs.o -ltre cssdiff.o -o cssdiff
 
-cssmerge: cssmerge.c crm_util_errorhandlers.o $(HFILES)
+cssmerge: cssmerge.c crm_str_funcs.o crm_util_errorhandlers.o $(HFILES)
 	$(CC)  $(CFLAGS) $(LDFLAGS) -D$(SYSTEMTYPE) -DVERSION='"$(VERSION)"' \
 		crm_str_funcs.o \
 		crm_util_errorhandlers.o \
